tfjs-backend-webgpu/src/kernels/webgpu_program.ts

```diff
@@ -73,7 +73,8 @@ export const compileProgram =
 
       const source = shader_preprocessor.makeShader(
           inputsData, outputData, program, isFromPixel);
-      const module = device.createShaderModule({code: source});
+      const module = device.createShaderModule(
+          {code: source, label: program.constructor.name});
       const pipeline = device.createComputePipeline(
           {layout: pipelineLayout, compute: {module, entryPoint: 'main'}});
```

Comment:
label for `createComputePipeline` is also needed?

Reply:
Pass this twice seems wired. But It seems createComputePipeline doesn't get this label from shader module.
Should we change createComputePipeline in dawn?

Reply:
That's a good point. Please file a bug to dawn to ask whether `createComputePipeline` can reuse `module`'s label if the label itself is not specified.

Reply:
Bug is here: https://bugs.chromium.org/p/dawn/issues/detail?id=1211